cli/config/configs.go

```diff
@@ -16,7 +16,8 @@ var ErrClientConfigNotFound = errors.New(heredoc.Doc(`
 `))
 
 type ClientConfig struct {
-	Host string `yaml:"host" cmdx:"host"`
+	Host     string `yaml:"host" cmdx:"host"`
+	UseHTTPS bool   `yaml:"use_https" cmdx:"use_https"`
```

Comment:
what if we call it `TLS` instead of `HTTPS`

Reply:
this reflects the schema whitelist in the swagger client. also, user is expected to set `host` config without `http://` or `https://` scheme prefix. so kept this as `use_https` to make it more obvious that it's going to use `https://` or not for calls.